flask_sqlalchemy/__init__.py

```diff
@@ -184,7 +184,7 @@ def __init__(self, db, autocommit=False, autoflush=True, **options):
             bind=bind, binds=binds, **options
         )
 
-    def get_bind(self, mapper=None, clause=None, bind=None):
+    def get_bind(self, mapper=None, clause=None, **kwargs):
```

Comment:
Since our function doesn't use `clause`, we can remove that to be handled by `kwargs`.